bazel/cargo.config

```diff
@@ -1 +1,2 @@
 [patch.crates-io]
+bitcoin = { git = "https://github.com/dfinity/rust-dogecoin", rev = "f1f900da0b193c43fd16f4598e614ed3eafbfd8f" }
```

Comment:
This patch (also in Cargo.toml) is necessary to override the version of rustbitcoin used by a dependency bitcoincore-rpc. Also, for the same reason, the rust-bitcoin version has to be set to an official version like `0.35.99` instead of a pre-release like `0.35.5-doge.0`.

This will not be necessary once we switch bitcoincore-rpc to using the same rust-bitcoin pre-release version (in the next PR).